Sources/PrimerSDK/Classes/Error Handler/Error.swift

```diff
@@ -114,6 +115,13 @@ enum KlarnaException: PrimerErrorProtocol {
                                      bundle: Bundle.primerResources,
                                      value: "Failed to find Klarna order items",
                                      comment: "Failed to find Klarna order items - Error message")
+            
+        case .orderItemMissesAmount:
+            return NSLocalizedString("primer-klarna-error-message-missing-order-item-amount",
+                                     tableName: nil,
+                                     bundle: Bundle.primerResources,
+                                     value: "Failed to find amount for an order item",
```

Comment:
[Q]
Is there any way this message gets displayed to the user? A user can't do anything with this 😅

Reply:
Same questions for the pending true below

Reply:
No, this error is the error thrown to the developer. Currently there's no alert showing from the SDK at all, we removed them a couple of weeks back.

Reply:
@ceriksson: this type of error shouldn't be given to any user, as it doesn't make any sense for them.
@EvansPie: since this error is and will only be for developers,  can we remove it from the localised string?

Reply:
It's common on iOS errors to be localized. They do have a property `localizedDescritpion`, which gives you the error description.

Reply:
Fair enough. It's a bit weird to me to take some time to translate developer error x) Let's keep it as-is for now